### PR TITLE
Tweak rendering parameters

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
retest this please

**Comment by eile** [comment]
retest this please

**Comment by eile** [comment]
@bilgili Please retrigger. I still can't.

**Comment by bilgili** [comment]
retest this please

**Comment by eile** [comment]
As an incentive, this PR should also fix the ZeroEQ connection broker test failure.

**Comment by bilgili** [comment]
Upstream tests are failing, why are we running them ?

**Comment by eile** [comment]
> why are we running them ?

Because they shouldn't (and didn't) fail. Murphy strikes again, I guess because the tests get executed more often now.

**Comment by eile** [comment]
retest this please

**Comment by bilgili** [comment]
Yeah but we trust software ! )

**Comment by eile** [comment]
I don't. ;)

**Comment by bilgili** [comment]
But, other software testing in a local software seems a bit fuzzy to me. Like testing jpeg library in GIMP.

**Comment by eile** [comment]
It's our software. We should finally fix the tests to run reliable.

**Comment by eile** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by eile** [comment]
Please review

**Comment by eile** [comment]
ping

File: livre/core/render/View.cpp [hunk 1]
```diff
@@ -54,38 +54,17 @@ const Viewport& View::getViewport() const
     return viewport_;
 }
 
-void View::render( const FrameInfo& frameInfo,
-                   const RenderBricks& renderBricks,
+void View::render( const FrameInfo& frameInfo, const RenderBricks& bricks,
```
Comment (bilgili) [inline livre/core/render/View.cpp]: keep as before please

File: livre/eq/Channel.cpp [hunk 2]
```diff
@@ -147,16 +148,11 @@ SelectVisibles::SelectVisibles( DashTreePtr dashTree,
...
-                       screenSpaceError,
-                       worldSpacePerVoxel,
-                       minLOD,
-                       maxLOD )
-      , _frustum( frustum )
-      , _volumeDepth( volumeDepth )
-{
-
-}
+    , _lodEvaluator( windowHeight, screenSpaceError, worldSpacePerVoxel,
```
Comment (bilgili) [inline livre/eq/Channel.cpp]: keep as before pelase, I am keeping for readability

File: livre/eq/Channel.cpp [hunk 3]
```diff
@@ -303,15 +297,9 @@ class Channel
...
-                                _currentFrustum,
-                                pixelViewport[3],
-                                screenSpaceError,
-                                worldSpacePerVoxel,
-                                volumeDepth,
-                                minLOD,
-                                maxLOD );
-
+        SelectVisibles visitor( dashTree, _currentFrustum, pixelViewport[3],
+                                screenSpaceError, worldSpacePerVoxel,
```
Comment (bilgili) [inline livre/eq/Channel.cpp]: ditto

**Pull request merged**